src/views/webview-app/components/connect-helper/connect-helper.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from 'react';
+import classnames from 'classnames';
+import { connect } from 'react-redux';
+import {
+  ActionTypes,
+  LinkClickedAction,
+  OpenConnectionStringInputAction,
+  SetCurrentViewAction
+} from '../../store/actions';
+import { WEBVIEW_VIEWS } from '../../extension-app-message-constants';
+
+const styles = require('./connect-helper.less');
+
+type dispatchProps = {
+  onLinkClicked: (screen: string, linkId: string) => void;
+  onOpenConnectionStringInput: () => void;
+  onOpenConnectionFrom: () => void;
+};
+
+type props = dispatchProps;
+
+function getOSCommandShortcutName(): string {
+  if (navigator.userAgent.indexOf('Win') != -1) {
+    return 'Ctrl';
+  }
+
+  return 'Cmd';
+}
+
+class ConnectHelper extends React.Component<props> {
+  render(): React.ReactNode {
+    const {
+      onOpenConnectionFrom,
+      onOpenConnectionStringInput
+    } = this.props;
+
+    return (
+      <div className={styles['connect-helper']}>
+        <div className={styles['connect-helper-connect-area']}>
+          <div className={classnames(styles['connect-helper-connect-option'], styles['connect-connection-string-area'])}>
+            <div>
+              Connect with
+            </div>
+            <div>
+              <strong>Connection String</strong>
+            </div>
+            <button
+              className={classnames(
+                styles['connect-helper-connect-option-button'],
+                styles['connect-helper-connection-string-button']
+              )}
+              onClick={() => onOpenConnectionStringInput()}
+            >
+              Connect
+            </button>
+          </div>
+          <div className={classnames(
+            styles['connect-helper-connect-option'],
+            styles['connect-connection-form-area']
+          )}>
+            <div>
+              Advanced
+            </div>
+            <div>
+              <strong>Connection Settings</strong>
+            </div>
+
+            <button
+              className={classnames(
+                styles['connect-helper-connect-option-button'],
+                styles['connect-helper-connection-form-button']
+              )}
+              onClick={() => onOpenConnectionFrom()}
+            >
+              Open form
+            </button>
+          </div>
+        </div>
+        <div className={styles['connect-helper-message']}>
+          <strong>{getOSCommandShortcutName()} + Shift + P</strong> for all MongoDB Command Palette options
```

Comment:
isn't there an api to just get the keybinding for the palette?

Reply:
I thought so, but I couldn't find the hotkeys on the configuration object https://code.visualstudio.com/api/references/vscode-api#WorkspaceConfiguration  `vscode.workspace.getConfiguration()` so I just went with showing the default :/